main.go

```diff
@@ -106,15 +106,7 @@ func main() {
 	service.Version = version
 	service.GitSHA = gitsha
 
-	// Register workloads
-	if err := oam.RegisterWorkloads(mesheryServerAddress(), serviceAddress()+":"+service.Port); err != nil {
-		log.Info(err.Error())
-	}
-
-	// Register traits
-	if err := oam.RegisterTraits(mesheryServerAddress(), serviceAddress()+":"+service.Port); err != nil {
-		log.Info(err.Error())
-	}
+	go registerCapabilities(service.Port, log)
```

Comment:
retries over here @utkarsh-pro ?

Reply:
@kumarabd retries are present in meshey-adapter-library